server/src/db/model.ts

```diff
@@ -21,6 +30,17 @@ const userSchema = new mongoose.Schema({
         height: { type: Number, required: true },
         fitnessLevel: { type: String, required: true },
     },
+    //using this field if the user is given the choice to skip the inetrest form
+    seen_greeting_modal: { type: Boolean, default: false },
+
+    // Exercise: [{ type: mongoose.Types.ObjectId, ref: 'Exercise' }],
+    exercises: [{
```

Comment:
instead of placing the entire schema here you could just write `[exerciseSchema]` and it would work